### PR TITLE
Remove unused parameter for macro in v_ext_macro.h

### PR DESCRIPTION
**Pull request by liweiwei90**
BODY8/16 and CHECK8/16 are unused in these cases.

### CODE REVIEW

File: riscv/insns/vfncvt_f_f_w.h [hunk 1]
```diff
@@ -1,9 +1,7 @@
-// vfncvt.f.f.v vd, vs2, vm
+// vfncvt.f.f.w vd, vs2, vm
```
Comment (aswaterman) [inline riscv/insns/vfncvt_f_f_w.h:1]: The comment changes can be a separate commit.

File: riscv/insns/vfncvt_f_f_w.h [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
 VI_VFP_NCVT_FP_TO_FP(
-  {;},                                 // BODY16
```
Comment (aswaterman) [inline riscv/insns/vfncvt_f_f_w.h:3]: Deleting BODY16 suggests that the PR description is incomplete.  I guess BODY16 is removed for narrowing ops and BODY8 is removed for single-width ops?

**Review by aswaterman — CHANGES_REQUESTED** [review]
Seems like a good improvement, modulo my comments.

File: riscv/insns/vfncvt_f_f_w.h (hunk 1 above)
Comment (liweiwei90) [reply riscv/insns/vfncvt_f_f_w.h:1]: OK. I have updated it.

File: riscv/insns/vfncvt_f_f_w.h (hunk 2 above)
Comment (liweiwei90) [reply riscv/insns/vfncvt_f_f_w.h:3]: OK. I have udpated the description and add a new commit to update the parameters for VI_VFP_NCVT_* macros to be consistent with these comments.

**Review by aswaterman — APPROVED** [review]
Thanks!
